meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -157,7 +169,8 @@ IMAGE_CMD_resinos-img () {
         ${RESIN_IMAGE_ALIGNMENT} \+ \
         ${RESIN_STATE_SIZE_ALIGNED} \+ \
         ${RESIN_IMAGE_ALIGNMENT} \+ \
-        ${RESIN_DATA_SIZE_ALIGNED} \
+        ${RESIN_DATA_SIZE_ALIGNED} \+ \
+        ${DEVICE_SPECIFIC_SPACE} \
```

Comment:
Can we have this ordered by location on disk? This would move the DEVICE_SPECIFIC_SPACE at the start of it.